apps/codeforafrica/turbo.json

```diff
@@ -6,7 +6,7 @@
     },
     "build-next": {
       "outputs": [".next/**", "!.next/cache/**", "dist/**"],
-      "env": []
+      "env": ["NEXT_PUBLIC_APP_NAME", "NEXT_PUBLIC_APP_URL"]
```

Comment:
Is this related to **this** PR?

Reply:
@kilemensi No, It is caused by the merged PR #583. I did not add it there.